composer.json

```diff
@@ -1,29 +1,31 @@
 {
     "name": "olcs/olcs-selfserve",
     "description": "OLCS Self Service Web Site",
-    "repositories": [
-        {
-            "packagist": false
-        },
-        {
-            "url": "https://nexus.olcs.dev-dvsacloud.uk/repository/OLCS",
-            "type": "composer"
-        },
-        {
-            "url": "https://nexus.olcs.dev-dvsacloud.uk/repository/Packagist-Proxy",
-            "type": "composer"
-        }
-    ],
     "require": {
         "php": ">=7.4",
         "dvsa/laminas-config-cloud-parameters": "^0.1.0",
+        "laminas/laminas-authentication": "^2.6",
+        "laminas/laminas-eventmanager": "^2.6",
+        "laminas/laminas-form": "^2.13",
+        "laminas/laminas-http": "^2.8",
+        "laminas/laminas-mvc": "^2.7",
+        "laminas/laminas-servicemanager": "^2.7",
+        "laminas/laminas-session": "^2.8",
+        "laminas/laminas-view": "^2.11",
+        "olcs/olcs-auth": "^5.0@alpha",
+        "olcs/olcs-common": "^4.206.516",
         "olcs/olcs-frontend-deps": "~4.206.0",
-        "olcs/olcs-laminas": "~2.5.0"
+        "olcs/olcs-logging": "^5.0@alpha",
+        "olcs/olcs-transfer": "^5.0@alpha"
     },
     "require-dev": {
+        "bamarni/composer-bin-plugin": "^1.8",
+        "johnkary/phpunit-speedtrap": "^4.0",
         "laminas/laminas-developer-tools": "*",
-        "olcs/olcs-devtools": "~4.206.0",
+        "laminas/laminas-modulemanager": "^2.9",
```

Comment:
I'd be surprised if this was a dev dependency. Can you double check this please?

Reply:
done , removed